creator-node/src/services/prometheusMonitoring/prometheus.constants.js

```diff
@@ -0,0 +1,56 @@
+const promClient = require('prom-client')
+const _ = require('lodash')
+
+/**
+ * For explanation of Metrics, and instructions on how to add a new metric, please see `prometheusMonitoring/README.md`
+ */
+
+// We add a namespace prefix to differentiate internal metrics from those exported by different exporters from the same host
+const AudiusPrefix = 'audius_'
+const DefaultPrefix = 'default_'
```

Comment:
If we're going to do a top-level namespace, I'd vote for `promclient_` to not clash with any default metrics the Prometheus Python client would produce if we used a similar namespacing ideology within discovery-provider.

Fwiw, we can't currently use custom collectors since we run our Python code in [multiprocess mode](https://github.com/prometheus/client_python#multiprocess-mode-eg-gunicorn). Hopefully support is added in the future for these default metrics.

Reply:
yep, changed this to actually prefix `audius_cn_default_` per your other comment, hopefully solves thsi